app/views/devise/mailer/confirmation_instructions.html.slim

```diff
@@ -1,4 +1,4 @@
-p.lead= t('mailer.confirmation_instructions.header', intro: @first_sentence)
+p.lead == t('mailer.confirmation_instructions.header', intro: @first_sentence)
```

Comment:
Yeah seems like this one could go either way, if you feel strongly about it I can update it, but we are currently using the `==` pattern in a few of those emails already so I thought it made most sense to just follow the convention that was already there.